ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -459,16 +459,30 @@
 }
 else
 {
+    string cssFileName = String.IsNullOrEmpty(Model.Puzzle.CustomCSSFile) ? "" : Model.Puzzle.CustomCSSFile.Trim();
+    @if (cssFileName.Length > 0)
+    {
+        // Add class to body wrapper here so only this page's content gets it, not every page
+        <script type="text/javascript">
+            document.addEventListener("DOMContentLoaded", () => {
+                let bodyWrapper = document.querySelector(".body-content");
+                bodyWrapper.classList.add("body-content-customizable");
+            });
+        </script>
+        cssFileName += (!cssFileName.EndsWith(".css")) ? ".css" : "";
```

Comment:
Does this work around something? I think it's reasonable to require correct file extensions

Reply:
This is just dummy proofing.  All you need to supply is a file name, so forcing users to remember to also add the extension seems unnecessarily complicated.  Plus, the worst that will happen if you get it wrong is that is just won't load because it won't be able to find the file.